nilearn/plotting/js_plotting_utils.py

```diff
@@ -120,8 +120,8 @@ def get_iframe(self, width=None, height=None):
         if height is None:
             height = self.height
```

Comment:
The python ternary operator equivalent can be used here:
```
height = height if height else self.height
```

Reply:
good to know!

This code is outside this PR.